src/components/CCard.vue

```diff
@@ -28,20 +28,23 @@
       {{ title }}
     </v-card-title>
 
-    <v-divider v-if="scrollable && title" class="mb-0"></v-divider>
+    <v-card-subtitle
+      v-if="subtitle"
+      class="c-card-subtitle px-6 px-md-8 text-wrap mb-3"
```

Comment:
Why the additional margin? Is ths related to the commit?

Reply:
... the new margin is added here instead